wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1296,6 +1297,61 @@ def workflow_status_with_date(workflow_state):
     return _("%(status_display)s %(task_name)s %(started_at)s") % translation_context
 
 
+@register.inclusion_tag(
+    "wagtailadmin/shared/keyboard_shortcuts_dialog.html", takes_context=True
+)
+def keyboard_shortcuts_dialog(context):
+    user_agent = context["request"].headers.get("User-Agent", "")
+    is_mac = re.search(r"Mac|iPod|iPhone|iPad", user_agent)
+    modifier = "⌘" if is_mac else "Ctrl"
+    shortcuts = {
```

Comment:
Sorry for the back and forth on structure here but a few changes I will do when merging but wanted to explain.

1. Let's use tuples for the shortcuts ->  `{"label": _("Copy"), "shortcut": f"{modifier} + c"}` would be `(_("Copy"), f"{modifier} + c")`. This is a bit easier to read when looking at a lot of these.
2. Let's stick to the same casing for `Shift` not a mix of `Shift` and `shift`.
3. Let's stick to the same casing for titles and labels, e.g. `Common actions` not `Common Actions` and `Paste and match style` not `Paste and Match Style`. Leaning towards 'Sentence case' not 'Title Case'.
4. Might be nice to add a simple docstring on this class for explanation.

```
    """
    Renders the keyboard shortcuts dialog content with the
    appropriate shortcuts for the user's platform.
    Note: Shortcut keys are intentionally not translated.
    """
```